### ISSUE TITLE
[Link] aria-label for external link icon is falling back to 'en' for multiple locales

### ISSUE THREAD

**Issue by francinen** [Shopify/polaris-react#1663]
## Issue summary
The `aria-label` for the external link icon is still rendering the English translation when the user's locale is set to any of the following non-English locales:
- `de`
- `es`
- `fr`
- `it`
- `ja`
- `nl`
- `pt-BR`
- `zh-CN`
- `zh-TW`

![image](https://user-images.githubusercontent.com/8534230/59306409-282d7300-8c6a-11e9-81b8-be6620bf95f7.png)

![image](https://user-images.githubusercontent.com/8534230/59306219-d4228e80-8c69-11e9-8ccd-401c03e2487b.png)

## Expected behavior
The `aria-label` should be set to a string that has been translated for the given locale.

## Steps to reproduce the problem

1. Render an external link on a page and set your locale to a value other than `en`

2. Inspect the external link icon and locate its `aria-label`. It should still be set to an English string.

## Reduced test case

The best way to get your bug fixed is to provide a reduced test case. This [CodeSandbox template](https://codesandbox.io/s/q82mlq0m26
) is a great starting point.

## Specifications

- Are you using the React components? (Y/N): Y
- Polaris version number: 3.16.0
- Browser: Chrome
- Device: MacBook Pro
- Operating System: Mojave 10.14.5

Or run `npx envinfo --system --binaries --browsers --npmPackages react,react-dom,@shopify/polaris` to provide specifications on your environment including version numbers, browser, device, and operating system.

Paste the results here:

```bash

```

**Comment by chloerice** [comment]
Hmmm, I think it's because the translation doesn't exist in `web`. I'll put up a PR.
